src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -45,19 +51,35 @@ final class ManagingProductsContext implements Context
     private $sharedStorage;
 
     public function __construct(
+        ApiClientInterface $adminUserClient,
         ApiClientInterface $client,
         ApiClientInterface $productReviewsClient,
```

Comment:
Order should be changed as the $client is the default one for this context, I wrote it in review yesterday 😢
And here you can see that there is already some convention for naming api client services, but you added it differently..